app/views/candidate_interface/personal_details/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title, page_title('personal_details') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <fieldset class="govuk-fieldset">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8">
+        <h1 class="govuk-fieldset__heading">
+          <%= t('page_titles.personal_details') %>
+        </h1>
+      </legend>
+
+      <%= form_with model: @personal_details_form, url: candidate_interface_personal_details_update_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+        <%= f.govuk_error_summary %>
+
+        <%= f.govuk_text_field :first_name, label: { text: t('application_form.personal_details.first_name.label'), size: 'm' }, hint_text: t('application_form.personal_details.first_name.hint_text') %>
+
+        <%= f.govuk_text_field :last_name, label: { text: t('application_form.personal_details.last_name.label'), size: 'm' }, hint_text: t('application_form.personal_details.last_name.hint_text') %>
+
+        <%= f.govuk_date_field :date_of_birth, date_of_birth: true, legend: { text: t('application_form.personal_details.date_of_birth.label'), tag: 'span' }, hint_text: t('application_form.personal_details.date_of_birth.hint_text') %>
+
+        <%= f.govuk_text_field :nationality, label: { text: t('application_form.personal_details.nationality.label'), size: 'm' } %>
+
+        <%= f.govuk_collection_radio_buttons :english_main, @personal_details_form.main_language_english_options, :id, :name, legend: { text: t('application_form.personal_details.main_language_english.label') } %>
+
+        <%= f.submit 'Continue', class: 'govuk-button' %>
```

Comment:
I think we can use `f.govuk_submit` here, so we don't have to specify `govuk-button`.